### PR TITLE
Feature/bocm 223 endpoint to get lpas

### PR DESCRIPTION
**Pull request by mayya-bond-kc**
## Describe your changes

Creates new endpoint to get list of LPAs from external endpoint

## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [x] I have performed a self-review of my own code
- [x] I have double checked this work does not include any hardcoded secrets or passwords
- [x] I have made corresponding changes to the documentation
- [x] I have provided details on how I have tested my code
- [x] I have referenced the ticket number above
- [x] I have provided a description of the ticket
- [x] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: apps/api/src/server/app/validation/validation.controller.js [hunk 1]
```diff
@@ -86,4 +88,14 @@ async function getAppealForValidation(appealId) {
 	return appeal;
 }
 
-export { getAppeals, getAppealDetails, updateAppeal, submitValidationDecision };
+const obtainLPAList = async function() {
+	const dataRaw = await got.get('https://services1.arcgis.com/ESMARspQHYMw9BZ9/arcgis/rest/services/LPA_APR_2021_UK_NC/FeatureServer/0/query?where=1%3D1&outFields=LPA21NM&outSR=4326&f=json');
```
Comment (radum) [inline apps/api/src/server/app/validation/validation.controller.js:92]: What happen if this request fails?

**Review by radum — APPROVED** [review]

File: apps/api/src/server/app/validation/validation.controller.js (hunk 1 above)
Comment (mayya-bond-kc) [reply apps/api/src/server/app/validation/validation.controller.js:92]: 500 with the error message of failure, can add to test

**Review by radum — APPROVED** [review]

**Pull request merged**